src/styles/product-card.scss

```diff
@@ -0,0 +1,105 @@
+.product-card {
+  width: 200px;
+  margin-right: 48px;
+  padding-bottom: 39px;
+
+  &__container {
+    width: 200px;
+    border: 1px solid $color-border-secondary;
+    border-radius: 5px;
+    background-color: $color-bg;
+  }
+
+  &__product-image {
+    width: 160px;
+    height: 134px;
+    padding-top: 32px;
+    margin: 0 19px;
+    display: flex;
+  }
+
+  &__description {
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-txt-primary;
+    width: 166px;
+    height: 36px;
+    margin: 40px 16px 4px;
+  }
+
+  &__product-code-description {
+    margin: 0 16px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    height: 14px;
+    color: $color-txt-secondary;
+  }
+
+  &__review-block {
+    display: flex;
+    justify-content: space-between;
+    height: 32px;
+    margin: 0 16px;
+    width: 166px;
+  }
+
+  &__info-description {
+    align-self: flex-end;
+    font-size: 10px;
+    margin: 0;
+    color: $color-txt-primary;
+    padding-bottom: 2px;
+  }
+
+  &__price-block {
+    margin: 0 16px;
+    display: flex;
+    justify-content: space-between;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    height: 42px;
+  }
+
+  &__price-title {
+    align-self: flex-end;
+    color: $color-txt-secondary;
+  }
+
+  &__price-quantity {
+    color: $color-txt-primary;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 16px;
+    align-self: flex-end;
+    margin-right: 2px;
+  }
+
+  &__button-action-link {
+    background-color: $color-link;
+    border: 1px solid transparent;
+    border-radius: 5px;
+    color: $color-bg;
+    text-transform: uppercase;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-decoration: none;
+    padding: 10px 0;
+    margin: 16px;
```

Comment:
you shouldnt give a margin to each element here, you can just set padding on the container to create this space